src/components/HomePage/HomePage.jsx

```diff
@@ -31,9 +34,11 @@ class HomePage extends React.Component {
     		linkColor: color
     	});
     }
+
     render() {
         return (
         	<div>
+						<SiteMask />
```

Comment:
Since we've switched over to using a video version of the static for performance issues, and videos don't autoplay on mobile (https://www.aerserv.com/why-does-video-autoplay-on-mobile-devices-not-work/), can we set it up so that the static doesn't play on mobile.

It'll means we have to address the color flicker issue for real, but that's probably the most responsible thing to do anyways haha.